app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -938,6 +938,11 @@ private void closeFullScreen() {
 
   public void showHelpPage() {
     getCurrentWebView().loadUrl("file:///android_asset/help.html");
+    /*
+     * Passing null to update the title to "Kiwix" with
+     * appropriate translations
+     */
```

Comment:
I think it's very obvious from the name of the method, what we want to achieve. I would suggest removing these comments or atleast make it one liner.

Reply:
I guess these comments should be kept because we're passing null to updateTitle which could be troublesome to understand for new developers. 
I guess I'll shorten the comments here and remove the comments in openMainPage would that be okay ? Do let me know your opinion. Thanks

Reply:
Okay, make it one liner using ```//```